File: Assignment36_5.c
// Write a program which accept string from user and count no of white spaces
#include<stdio.h>

int Count(char *str)
{
    int iCount = 0;
    
    while(*str != '\0')
    {
        if(*str == ' ')
        {
            iCount++;
            
        }
        str++;
    }

    return iCount;
}

int main()
{
    char Arr[] = {'\0'};
    int iRet = 0;

    printf("Enter a String: \n");
    scanf("%[^'\n']s",Arr);

    iRet = Count(Arr);
    
    printf("Count of White spaces are: %d\n",iRet);

    return 0;
}

File: Assignment36_5.java
// Write a program which accept string from user and count no of white spaces
import java.util.*;

class Logic
{
    public int CountWhiteSpace(String str)
    {
        char Arr[] = str.toCharArray();
        int i = 0;
        int iCount = 0;

        for(i = 0; i < Arr.length; i++)
        {
            if(Arr[i] == ' ')
            {
                iCount++;
            }            
        }
        return iCount;
    }
}


class Assignment36_5
{
    public static void main(String A[])
    {
        Scanner scanobj = new Scanner(System.in);

        System.out.println("Enter a string: ");
        String sobj = scanobj.nextLine();

        Logic lobj = new Logic();
        int iRet = lobj.CountWhiteSpace(sobj);
        System.out.println("Number of white spaces are: "+iRet);
    }
}